### PR TITLE
8 feat add about page

### PR DESCRIPTION
**Pull request by Stefan-Radu**
Each of us should update the config with the appropriate details.

### CODE REVIEW

**Comment by Mathis-Dory** [comment]
I will do it now :)

**Comment by Costinteo** [comment]
@Stefan-Radu you forgot to add your line lol.

**Comment by Stefan-Radu** [comment]
@Costinteo to link your github xD

**Comment by Mathis-Dory** [comment]
I added my part, I think @SecioreanuStefanita  you are the last one and then we can merge the PR.

**Review by Mathis-Dory — APPROVED** [review]

File: config.yaml [hunk 1]
```diff
@@ -17,6 +17,37 @@ params:
...
+    - name: zenbassi
+      link: https://github.com/stefan-radu
+      picture: https://avatars.githubusercontent.com/u/30214214?v=4
+      tags: [ 'forensics','pwn' ,'rev', 'misc' ]
+      description:
+        - likes vim a bit too much
+        - into (film) photography
+    - name: Koyossu
+      link: https://github.com/SecioreanuStefanita
+      picture: https://avatars.githubusercontent.com/u/39965799?v=4
```
Comment (Mathis-Dory) [inline config.yaml:37]: Change it @SecioreanuStefanita

**Review by Mathis-Dory — CHANGES_REQUESTED** [review]

**Pull request merged**